src/harness/testcases/WINNF_FT_S_DRG_testcase.py

```diff
@@ -269,8 +272,175 @@ def test_WINNF_FT_S_DRG_5(self):
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+
+    # Create the actual config.
+    devices = [device_a, device_c, device_b]
+    grant_requests = [grant_a, grant_c, grant_b]
+    heartbeat_requests = [heartbeat_a, heartbeat_c, heartbeat_b]
+    deregistration_requests = [deregister_a, deregister_c, deregister_b]
+    config = {
+        'registrationRequest': devices,
+        'conditionalRegistrationData': conditionals,
+        'grantRequest': grant_requests,
+        'heartbeatRequest': heartbeat_requests,
+        'deregistrationRequest': deregistration_requests,
+        # First request has an invalid CBSD ID => INVALID_VALUE
+        # Second request is missing CBSD ID => MISSING_PARAM
+        # Third request, all valid params => SUCCESS
+        'expectedResponseCodes': [(103,), (102,), (0,)]
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_DRG_6_default_config)
+  def test_WINNF_FT_S_DRG_6(self, config_filename):
+    """[Configurable] Array Deregistration."""
+
+    config = loadConfig(config_filename)
+    # Very light checking of the config file.
+    self.assertEqual(
+        len(config['registrationRequest']), len(config['grantRequest']))
+    self.assertTrue(
+        len(config['grantRequest']), len(config['heartbeatRequest']))
+    self.assertTrue(
+        len(config['heartbeatRequest']), len(config['deregistrationRequest']))
+    self.assertEqual(
+        len(config['deregistrationRequest']),
+        len(config['expectedResponseCodes']))
+
+    # Whitelist FCC IDs.
+    for device in config['registrationRequest']:
+      self._sas_admin.InjectFccId({
+          'fccId': device['fccId'],
+          'fccMaxEirp': 47
+      })
+
+    # Whitelist user IDs.
+    for device in config['registrationRequest']:
+      self._sas_admin.InjectUserId({'userId': device['userId']})
+
+    # Step 2 & 3: Register devices and get grants
+    cbsd_ids, grant_ids = self.assertRegisteredAndGranted(
+        config['registrationRequest'], config['grantRequest'],
+        config['conditionalRegistrationData'])
+
+    # Step 4: First Heartbeat Request
+    heartbeat_request = config['heartbeatRequest']
+    addCbsdIdsToRequests(cbsd_ids, heartbeat_request)
+    addGrantIdsToRequests(grant_ids, heartbeat_request)
+    request = {'heartbeatRequest': heartbeat_request}
+    responses = self._sas.Heartbeat(request)['heartbeatResponse']
+    self.assertEqual(len(responses), len(heartbeat_request))
+    # Check heartbeat response
+    for i, response in enumerate(responses):
+      self.assertEqual(response['response']['responseCode'], 0)
+    del request, responses
+
+    # Step 5: First deregistration request
+    degregister_request = config['deregistrationRequest']
```

Comment:
Typo in variable name   degregister_request

Reply:
Done.